### PR TITLE
Various changes

### PR DESCRIPTION
**Pull request by namjay1170**
ERP changes
-Added new reagent
-Made said reagent come into play during degenerecy
Drug nerf (test)
-Makes speedy drugs not make you cross the map in 0.0007 seconds
Added FEV
-Slightly better than unstable mutagen and adminspawn only

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -1193,3 +1193,16 @@ datum/reagent/medicine/super_stimpak/on_mob_life(mob/living/M)
...
+	name = "Cum"
+	id = "cum"
+	description = "Jesus, what the fuck is this?"
+	reagent_state = LIQUID
+	color = "#DFD7AF" // rgb: 223, 215, 175
+
+/datum/reagent/medicine/cum/on_mob_life(mob/living/M)
+	M.adjustFireLoss(-0.1*REM)
+	M.adjustBruteLoss(-0.1*REM)
+	.=..()
```
Comment (Qustinnus) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: good boy

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 2]
```diff
@@ -43,9 +43,32 @@
...
+		return  //No robots, AIs, aliens, Ians or other mobs should be affected by this.
+	if((method==VAPOR && prob(min(33, reac_volume))) || method==INGEST || method==PATCH || method==INJECT)
+		randmuti(M)
+		if(prob(75))
+			randmutb(M)
+		else
+			randmutg(M)
+		M.updateappearance()
+		M.domutcheck()
+	..()
```
Comment (Qustinnus) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: bad boy

Comment (namjay1170) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: Late night button pressing is always a gamble, 'tis fixed now

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 3]
```diff
@@ -1193,3 +1193,16 @@ datum/reagent/medicine/super_stimpak/on_mob_life(mob/living/M)
 	M.Dizzy(2)
 	..()
 	return
+
+/datum/reagent/medicine/cum
+	name = "Cum"
+	id = "cum"
+	description = "Jesus, what the fuck is this?"
```
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: agreed

Comment (Angryon) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: this pr makes me want to die

**Pull request merged**